packages/context/src/context-view.ts

```diff
@@ -152,3 +152,19 @@ export class ContextView<T = unknown> extends EventEmitter
     return () => this.values(session);
   }
 }
+
+/**
+ * Create a context view as a getter
+ * @param ctx Context object
+ * @param bindingFilter A function to match bindings
+ * @param session Resolution session
+ */
+export function createViewGetter<T = unknown>(
```

Comment:
+1 for making these `@loopback/context` changes in a standalone commit, so that release notes are populated with a meaningful entry.